lint/programming_utils.py

```diff
@@ -0,0 +1,53 @@
+import ast
+
+import gamla
+
+
+def _get_functions_and_references(node):
+    return gamla.pipe(
+        node,
+        gamla.case_dict(
+            {
+                gamla.is_instance(ast.Call): gamla.compose_left(
```

Comment:
I tried without it, and some attributes can have a call as a value. I suspect it might be due to the functional programming but It didn't work without recursion.